styles.css

```diff
@@ -1015,52 +1025,72 @@ footer hr {
     padding-top: 0;
   }
 
+  .featured .card-h {
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  #featured-p {
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  .card-link {
+    display: none;
+  }
+
   /* -------------------- */
 
   /* ABOUT ME PAGE */
   .aboutme {
     padding: 130px 142px 0 130px;
     background: url("assets/illustration-about-me.png") no-repeat;
     background-position: right;
-    background-size: 38%;
+    background-size: 40%;
     background-color: #1c1a19;
   }
 
   .aboutme h1 {
     text-align: left;
     font-size: 72px;
     line-height: 82px;
+    color: #f55800;
   }
 
   .aboutme p {
     font-size: 20px;
     line-height: 32px;
     width: 566px;
-    padding: 0 0 48px 0;
+    padding: 18px 0 48px 0;
+    margin: 0;
   }
 
   .get-resume {
     padding-bottom: 139px;
   }
 
+  div .see-project {
+    background: #f55800;
+  }
+
   .skills {
     display: flex;
     flex-direction: row;
     align-items: flex-start;
-    justify-content: space-between;
-    padding: 146px 141px;
+    gap: 210px;
+    padding: 130px 141px;
   }
 
   .skills ul {
     padding: 0;
     font-weight: 700;
     font-size: 32px;
-    line-height: 44px;
   }
 
   .skills li {
     text-align: left;
     padding: 12px 0;
+    line-height: 24px;
   }
 
   /*******************   */
```

Comment:
**[REQUIRED CHANGES]**

- Please, kindly ensure the black line is not showing from the min-width of 768px on your footer to match the Figma design below:

![REMOVE](https://user-images.githubusercontent.com/59397159/172700491-5d98dade-8a4c-4416-96b1-18ad8946c9d8.png)

Reply:
hi @zieeco, first of all, thanks for the review!

I have a question about the screen size because on Figma the screen size width was 1440p, so if I use the recommended font size things won't fit, that's why I was creating other media queries for different breakpoints in width. I think I got confused and thought I read somewhere the size was 1200px.

So how do I make it work to 768px so the texts reduce and things can fit?

Reply:
@Tiago-Lelinski-Marin, sorry for replying late! You could add as many breakpoints as possible but it shouldn't be lesser than 768px to meet the project requirement first. You are very correct as the Figma is being viewed on 1440px. The 1440px are for bigger screen size but do have in mind that the 1440px used in Figma was just used as a view [how the design looks on a larger screen not necessarily a breakpoint of 1440px]. To ensure your design is close to that on the Figma, you need to change the view to 1440px and compare your design if it's close to the design being viewed in the Figma design. 

![1440px](https://user-images.githubusercontent.com/59397159/172729218-e715e9f5-46f1-4b41-8f9b-f526eef25e88.png)

**NOTE: you could use any breakpoint of your choice but shouldn't be lesser than 768px**

Reply:
Thanks a lot! 

Your explanation was very clear and now I think I got it!

Reply:
@Tiago-Lelinski-Marin, I'm glad you've gotten a clue, you could still reach out if you get stuck. 
Keep rocking :dancers:  and coding :computer: 
Cheers